test/e2e/common_test.go

```diff
@@ -317,7 +317,7 @@ func (p *PodmanTestIntegration) createArtifact(image string) {
 	fmt.Printf("Caching %s at %s...", image, destName)
 	if _, err := os.Stat(destName); os.IsNotExist(err) {
 		pull := p.PodmanNoCache([]string{"pull", image})
-		pull.Wait(90)
+		pull.Wait(240)
```

Comment:
Was this a debugging change?

Reply:
No, there's another commit to bump it.

I was hitting this limit too frequently at home (low bandwitdh).  It's used to fetch the images for testing and since they're big, 90 seconds are not always enough :^)

Reply:
+1 I'm facing the same issue when running tests.